### PR TITLE
Azure FileCopy task V3Users niadak supportazmoduleininitializeazure

### PR DESCRIPTION
**Pull request by niadak**

### CODE REVIEW

File: Tasks/AzureFileCopyV3/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+### Prerequisite for the task
+
+**Azure Subscription**
+
+To deploy to Azure, an Azure subscription has to be linked to Team Foundation Server or to Azure Pipelines using the Services tab in the Account Administration section. Add the Azure subscription to use in the Build or Release Management definition by opening the Account Administration screen (gear icon on the top-right of the screen) and then click on the Services Tab. 
+
+ - For Azure Classic resources use 'Azure' endpoint type with Certificate or Credentials based authentication. If you are using credentials based auth, ensure that the credentials are for a [**work account**](https://azure.microsoft.com/en-in/pricing/member-offers/msdn-benefits-details/work-accounts-faq/) because Microsoft accounts like [**joe@live.com**](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/DeployAzureResourceGroup) or [**joe@hotmail.com**](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/DeployAzureResourceGroup) are not supported.
```
Comment (kmkumaran) [inline Tasks/AzureFileCopyV3/README.md]: I guess we will have to clean-up the  Readme given that we are removing classic support in latest version.

File: Tasks/AzureFileCopyV2/package-lock.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
       "resolved": "https://registry.npmjs.org/brace-expansion/-/brace-expansion-1.1.11.tgz",
       "integrity": "sha512-iCuPHDFgrHX7H2vEI/5xpz07zSHB00TpugqhmYtVmMO6518mCuRMoOYFldEBl0g187ufozdaHgWKcYFb61qGiA==",
       "requires": {
-        "balanced-match": "1.0.0",
+        "balanced-match": "^1.0.0",
```
[the remark this answers is not in the document]
Comment (vincentdass) [reply Tasks/AzureFileCopyV2/package-lock.json]: @niadak , Can you revert this file change, downgrade npm version to 5.6.0 and build the task.

File: Tasks/AzureFileCopyV3/task.loc.json [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+      "name": "ConnectedServiceNameARM",
+      "aliases": [
+        "azureSubscription"
+      ],
+      "type": "connectedService:AzureRM",
+      "label": "ms-resource:loc.input.label.ConnectedServiceNameARM",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "ms-resource:loc.input.help.ConnectedServiceNameARM",
+      "visibleRule": "ConnectedServiceNameSelector = ConnectedServiceNameARM"
```
Comment (niadak) [inline Tasks/AzureFileCopyV3/task.loc.json]: Remove this visibility rule

File: Tasks/Common/VstsAzureHelpers_/InitializeAzModuleFunctions.ps1 [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+function Set-CurrentAzSubscription {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$SubscriptionId,
+        [string]$TenantId)
+
+    $additional = @{ }
+    if ($TenantId) { $additional['TenantId'] = $TenantId }
```
[the remark this answers is not in the document]
Comment (niadak) [reply Tasks/Common/VstsAzureHelpers_/InitializeAzModuleFunctions.ps1]: yes. TenantId will be present

**Pull request merged**